components/org.wso2.carbon.identity.organization.management.service/src/test/java/org/wso2/carbon/identity/organization/management/service/OrganizationManagerImplTest.java

```diff
@@ -69,7 +68,6 @@
 import static org.wso2.carbon.identity.organization.management.service.constant.OrganizationManagementConstants.PATCH_PATH_ORG_DESCRIPTION;
 import static org.wso2.carbon.identity.organization.management.service.constant.OrganizationManagementConstants.PATCH_PATH_ORG_NAME;
 
-@WithAxisConfiguration
```

Comment:
Following compilation error has been reported after the carbon multitenancy version upgrade

```
[ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] /home/ashas/Desktop/workspace/public_github_repo/identity-organization-management-core/components/org.wso2.carbon.identity.organization.management.service/src/test/java/org/wso2/carbon/identity/organization/management/service/OrganizationManagerImplTest.java:[32,46] package org.wso2.carbon.identity.common.testng does not exist
[ERROR] /home/ashas/Desktop/workspace/public_github_repo/identity-organization-management-core/components/org.wso2.carbon.identity.organization.management.service/src/test/java/org/wso2/carbon/identity/organization/management/service/OrganizationManagerImplTest.java:[72,2] cannot find symbol
  symbol: class WithAxisConfiguration
```

To address the build error, `@WithAxisConfiguration` annotation is removed instead of adding the org.wso2.carbon.identity.testutil dependency, as adding the dependency could cause a cyclic dependency issue.  @WithAxisConfiguration annotation was not required for the current organization management tests as they do not interact with SOAP or Axis-based services.